src/cookbooks/glam.md

```diff
@@ -4,6 +4,8 @@ GLAM was built to help Mozillians answer their data questions without needing da
 
 GLAM is one of several high-level data tools that we provide at Mozilla. For more information, see [Tools for Data Analysis](../tools/interfaces.md).
 
+Access to GLAM is currently limited to Mozilla employees and designated contributors (this [may change in the future](https://bugzilla.mozilla.org/show_bug.cgi?id=1712353)). For more information, see [gaining access](../concepts/gaining_access.md).
+
 ## How to use GLAM
 
 You can visit GLAM at [`glam.telemetry.mozilla.org`](https://glam.telemetry.mozilla.org). As of this writing, Mozilla LDAP credentials are required to access it.
```

Comment:
With the above addition should we remove this part: 

> As of this writing, Mozilla LDAP credentials are required to access it.

Reply:
I can't believe I missed that! I think this is an argument for putting this information closer to the top. 😄